nix/modules/tezos-accuser.nix

```diff
@@ -8,7 +8,9 @@ with lib;
 let
   octez-accuser-pkgs = {
     "PtNairob" =
-      "${pkgs.octezPackages.octez-baker-PtNairob}/bin/octez-baker-PtNairob";
+      "${pkgs.octezPackages.octez-accuser-PtNairob}/bin/octez-accuser-PtNairob";
+    "Proxford" =
+      "${pkgs.octezPackages.octez-accuser-Proxford}/bin/octez-accuser-Proxford";
```

Comment:
This seems an odd change, how come it was a `baker` binary before and now it's an accuser? :thinking:

Reply:
it should be accuser (see file name), it was baker by mistake